arbiter-core/src/math/stochastic_process.rs

```diff
@@ -0,0 +1,251 @@
+pub enum StochasticProcessType {
+    /// Brownian motion
+    BrownianMotion(BrownianMotion),
+    /// Ornstein-Uhlenbeck
+    OrnsteinUhlenbeck(OrnsteinUhlenbeck),
+    /// Black-Derman-Toy
+    BlackDermanToy(BlackDermanToy),
+    /// Cox-Ingersoll-Ross
+    CoxIngersollRoss(CoxIngersollRoss),
+    /// Extended Vasicek
+    ExtendedVasicek(ExtendedVasicek),
+    /// Ho-Lee
+    HoLee(HoLee),
+    /// Hull-White
+    HullWhite(HullWhite),
+}
+/// Struct for all processes init parameters.
+pub struct EulerMaruyamaInput {
+    /// initial value at t_0
+    pub x_0: f64,
+    /// initial time point
+    pub t_0: f64,
+    /// terminal time point
+    pub t_n: f64,
+    /// number of time steps between t_0 and t_n
+    pub n_steps: usize,
+    /// how many process trajectories to simulate
+    pub m_paths: usize,
+    /// run in parallel or not (recommended for > 1000 paths)
+    pub parallel: bool,
+}
+
+/// Create new process and run euler maruyama.
+pub fn new_procces(
+    proccess_type: StochasticProcessType,
+    config: EulerMaruyamaInput,
+) -> Result<Trajectories> {
+    let trajectories: Trajectories = match proccess_type {
+        StochasticProcessType::BrownianMotion(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+        StochasticProcessType::OrnsteinUhlenbeck(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+        StochasticProcessType::BlackDermanToy(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+        StochasticProcessType::CoxIngersollRoss(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+        StochasticProcessType::ExtendedVasicek(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+        StochasticProcessType::HoLee(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+        StochasticProcessType::HullWhite(process) => process.euler_maruyama(
+            config.x_0,
+            config.t_0,
+            config.t_n,
+            config.n_steps,
+            config.m_paths,
+            config.parallel,
+        ),
+    };
+
+    Ok(trajectories)
+}
```

Comment:
I'm not sure we need this. Can you explain why we want to have this versus just doing something like:
```rust
pub use rustquant::some_module::SomeType;
```
in order to expose the crate to our end users?

We could also do:
```rust
pub type MyType = rustquant::some_module::ComplexOrUnclearTypeName;
```

Reply:
We could also do that. Do you think this is better?

Reply:
I think it reduces our codebase and gives us the same capabilities, so I'm in favor for the reduction.

I'd probably go for the `pub use` version and when we parse a `config.toml` (eventually), we can just have `x_0`, `t_0`, `t_n`, etc. as values in the `config.toml`.

Reply:
Okay i took it out and just made it a `pub use`